bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/NiriPrinter.java

```diff
@@ -52,44 +52,35 @@ private void writeSpectroscopyOutput(final UUID id, final SpectroscopyResult res
 
         _println("");
 
-        // This object is used to format numerical strings.
-        final FormatStringWriter device = new FormatStringWriter();
-        device.setPrecision(2); // Two decimal places
-        device.clear();
-
         // Altair specific section
         if (result.aoSystem().isDefined()) {
             _println(HtmlPrinter.printSummary((Altair) result.aoSystem().get()));
         }
 
         if (!result.observation().isAutoAperture()) {
-            _println("software aperture extent along slit = "
-                    + device.toString(result.observation().getApertureDiameter()) + " arcsec");
+            _println(String.format("software aperture extent along slit = %.2f arcsec", result.observation().getApertureDiameter()));
         } else {
             switch (result.source().getProfileType()) {
```

Comment:
Again, do we really need a `switch` here for two cases each with one statement and no default?